aws/provider.go

```diff
@@ -354,6 +354,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_api_gateway_usage_plan":                              resourceAwsApiGatewayUsagePlan(),
 			"aws_api_gateway_usage_plan_key":                          resourceAwsApiGatewayUsagePlanKey(),
 			"aws_api_gateway_vpc_link":                                resourceAwsApiGatewayVpcLink(),
+			"aws_api_gateway_v2_api":                                  resourceAwsApiGatewayV2Api(),
```

Comment:
Based on our naming standards at https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#new-service, the service name should be`apigatewayv2`, and therefore the resource should be `aws_apigatewayv2_api`.

Reply:
Sure. For completeness I'll rename the files to `resource_aws_api_gatewayv2_api.go`, `resource_aws_api_gatewayv2_api_test.go` and `api_gatewayv2_api.html.markdown`.

Reply:
Done.

Reply:
One more tweak, please: can you change it to `aws_apigatewayv2_api`, without the underscore in `apigateway`? We know it's not consistent with v1, but we're trying to standardize on our service naming.

Reply:
Too close to the issue to see the additional `_` 😄.
I have renamed the resource to `aws_apigatewayv2_api`.
Re-ran acceptance tests:

```console
$ make testacc TEST=./aws/ TESTARGS='-run=TestAccAWSAPIGatewayV2Api_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws/ -v -count 1 -parallel 20 -run=TestAccAWSAPIGatewayV2Api_ -timeout 120m
=== RUN   TestAccAWSAPIGatewayV2Api_basicWebSocket
=== PAUSE TestAccAWSAPIGatewayV2Api_basicWebSocket
=== RUN   TestAccAWSAPIGatewayV2Api_basicHttp
=== PAUSE TestAccAWSAPIGatewayV2Api_basicHttp
=== RUN   TestAccAWSAPIGatewayV2Api_disappears
=== PAUSE TestAccAWSAPIGatewayV2Api_disappears
=== RUN   TestAccAWSAPIGatewayV2Api_AllAttributesWebSocket
=== PAUSE TestAccAWSAPIGatewayV2Api_AllAttributesWebSocket
=== RUN   TestAccAWSAPIGatewayV2Api_AllAttributesHttp
=== PAUSE TestAccAWSAPIGatewayV2Api_AllAttributesHttp
=== RUN   TestAccAWSAPIGatewayV2Api_Tags
=== PAUSE TestAccAWSAPIGatewayV2Api_Tags
=== CONT  TestAccAWSAPIGatewayV2Api_basicWebSocket
=== CONT  TestAccAWSAPIGatewayV2Api_AllAttributesHttp
=== CONT  TestAccAWSAPIGatewayV2Api_Tags
=== CONT  TestAccAWSAPIGatewayV2Api_disappears
=== CONT  TestAccAWSAPIGatewayV2Api_AllAttributesWebSocket
=== CONT  TestAccAWSAPIGatewayV2Api_basicHttp
--- PASS: TestAccAWSAPIGatewayV2Api_disappears (21.75s)
--- PASS: TestAccAWSAPIGatewayV2Api_basicWebSocket (26.20s)
--- PASS: TestAccAWSAPIGatewayV2Api_basicHttp (38.05s)
--- PASS: TestAccAWSAPIGatewayV2Api_Tags (44.29s)
--- PASS: TestAccAWSAPIGatewayV2Api_AllAttributesHttp (74.49s)
--- PASS: TestAccAWSAPIGatewayV2Api_AllAttributesWebSocket (75.41s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	75.468s
```